src/foam/u2/images/account-structure-enabled.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's give this asset a more generic name, it's more just a "tree view" icon is it not?

Reply:
Done.